proxy-node-gateway/src/test/java/uk/gov/ida/notification/apprule/HubResponseAppRuleTests.java

```diff
@@ -153,13 +157,8 @@ private Response extractEidasResponse(Response hubResponse) throws Exception {
     }
 
     private javax.ws.rs.core.Response postHubResponseToProxyNode(Response hubResponse) throws URISyntaxException {
-        String encodedResponse = Base64.encodeAsString(marshaller.transformToString(hubResponse));
-        Form postForm = new Form().param(SamlFormMessageType.SAML_RESPONSE, encodedResponse);
-
-        return proxyNodeAppRule
-                .target("/SAML2/SSO/Response/POST")
-                .request()
-                .post(Entity.form(postForm));
+        postEidasAuthnRequest(eidasAuthnRequest);
```

Comment:
These tests now always test the happy path, where the gateway has seen an eIDAS authn request before it sees the corresponding hub response. It might be worth adding a test to check that we get the expected exception if this isn't the case.